<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Botonera</class>
 <widget class="QMainWindow" name="Botonera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>155</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menu Principal</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_username_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>198</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNuevo">
    <property name="title">
     <string>Archi&amp;vo</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionBorrame"/>
    <addaction name="actionConfigurar"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuAplicaciones">
    <property name="title">
     <string>Ap&amp;licaciones</string>
    </property>
    <addaction name="actionUsuarios"/>
    <addaction name="actionSoporte"/>
    <addaction name="actionSedes"/>
    <addaction name="actionMailing"/>
    <addaction name="actionVirtManager"/>
    <addaction name="actionActiveDirectory"/>
    <addaction name="actionISL"/>
   </widget>
   <widget class="QMenu" name="menuAcerca_de">
    <property name="title">
     <string>A&amp;yuda</string>
    </property>
    <addaction name="actionAcerca_de"/>
    <addaction name="actionMame"/>
   </widget>
   <widget class="QMenu" name="menuLink_Web">
    <property name="title">
     <string>Li&amp;nk Web</string>
    </property>
    <addaction name="actionBeiro"/>
    <addaction name="actionWebmail"/>
    <addaction name="actionCronos"/>
    <addaction name="actionGlpi"/>
    <addaction name="actionOcs"/>
   </widget>
   <addaction name="menuNuevo"/>
   <addaction name="menuAplicaciones"/>
   <addaction name="menuLink_Web"/>
   <addaction name="menuAcerca_de"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>70</width>
     <height>70</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionUsuarios"/>
   <addaction name="actionSoporte"/>
   <addaction name="actionSedes"/>
   <addaction name="actionLink"/>
   <addaction name="actionApp"/>
   <addaction name="actionActiveDirectory"/>
   <addaction name="actionConfigurar"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <action name="actionUsuarios">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../iconos.qrc">
     <normaloff>:/imagenes/iconos/botonera/usuarios.png</normaloff>:/imagenes/iconos/botonera/usuarios.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Usuarios</string>
   </property>
   <property name="toolTip">
    <string>Usuarios</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F1</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionCronos">
   <property name="icon">
    <iconset resource="../iconos.qrc">
     <normaloff>:/imagenes/iconos/botonera/cronos.png</normaloff>:/imagenes/iconos/botonera/cronos.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Cronos</string>
   </property>
   <property name="toolTip">
    <string>Cronos</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F2</string>
   </property>
  </action>
  <action name="actionWebmail">
   <property name="icon">
    <iconset resource="../iconos.qrc">
     <normaloff>:/imagenes/iconos/botonera/WebMail.png</normaloff>:/imagenes/iconos/botonera/WebMail.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Webmail</string>
   </property>
   <property name="toolTip">
    <string>Webmail</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F3</string>
   </property>
  </action>
  <action name="actionBeiro">
   <property name="icon">
    <iconset resource="../iconos.qrc">
     <normaloff>:/imagenes/iconos/botonera/beiro_cartera.png</normaloff>:/imagenes/iconos/botonera/beiro_cartera.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Beiro</string>
   </property>
   <property name="toolTip">
    <string>Beiro</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F4</string>
   </property>
  </action>
  <action name="actionGlpi">
   <property name="icon">
    <iconset resource="../iconos.qrc">
     <normaloff>:/imagenes/iconos/botonera/glpi.png</normaloff>:/imagenes/iconos/botonera/glpi.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Glpi</string>
   </property>
   <property name="toolTip">
    <string>Glpi</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F5</string>
   </property>
  </action>
  <action name="actionOcs">
   <property name="icon">
    <iconset resource="../iconos.qrc">
     <normaloff>:/imagenes/iconos/botonera/ocs.png</normaloff>:/imagenes/iconos/botonera/ocs.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Ocs</string>
   </property>
   <property name="toolTip">
    <string>Ocs</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F6</string>
   </property>
  </action>
  <action name="actionActiveDirectory">
   <property name="icon">
    <iconset resource="../iconos.qrc">
     <normaloff>:/imagenes/iconos/botonera/AD.png</normaloff>:/imagenes/iconos/botonera/AD.png</iconset>
   </property>
   <property name="text">
    <string>&amp;activeDirectory</string>
   </property>
   <property name="toolTip">
    <string>Active Directory</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F7</string>
   </property>
  </action>
  <action name="actionNuevo_Men">
   <property name="icon">
    <iconset resource="../iconos.qrc">
     <normaloff>:/imagenes/iconos/botonera/logo.png</normaloff>:/imagenes/iconos/botonera/logo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Nueva Botonera</string>
   </property>
  </action>
  <action name="actionISL">
   <property name="icon">
    <iconset resource="../iconos.qrc">
     <normaloff>:/imagenes/iconos/botonera/ISL.png</normaloff>:/imagenes/iconos/botonera/ISL.png</iconset>
   </property>
   <property name="text">
    <string>&amp;ISL</string>
   </property>
   <property name="toolTip">
    <string>ISLlight</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+9</string>
   </property>
  </action>
  <action name="actionConfigurar">
   <property name="icon">
    <iconset resource="../iconos.qrc">
     <normaloff>:/imagenes/iconos/botonera/update.png</normaloff>:/imagenes/iconos/botonera/update.png</iconset>
   </property>
   <property name="text">
    <string>Co&amp;nfigurar</string>
   </property>
  </action>
  <action name="actionSoporte">
   <property name="icon">
    <iconset resource="../iconos.qrc">
     <normaloff>:/imagenes/iconos/botonera/equipos.png</normaloff>:/imagenes/iconos/botonera/equipos.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Soporte</string>
   </property>
   <property name="toolTip">
    <string>Soporte</string>
   </property>
  </action>
  <action name="actionSedes">
   <property name="icon">
    <iconset resource="../iconos.qrc">
     <normaloff>:/imagenes/iconos/botonera/300px-Granada_-_Mapa_municipal.svg.png</normaloff>:/imagenes/iconos/botonera/300px-Granada_-_Mapa_municipal.svg.png</iconset>
   </property>
   <property name="text">
    <string>S&amp;edes</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="icon">
    <iconset resource="../iconos.qrc">
     <normaloff>:/imagenes/iconos/botonera/usuario.png</normaloff>:/imagenes/iconos/botonera/usuario.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Acerca de ...</string>
   </property>
  </action>
  <action name="actionAtalaya">
   <property name="icon">
    <iconset resource="../iconos.qrc">
     <normaloff>:/imagenes/iconos/botonera/atalaya.png</normaloff>:/imagenes/iconos/botonera/atalaya.png</iconset>
   </property>
   <property name="text">
    <string>Atalaya</string>
   </property>
   <property name="toolTip">
    <string>Atalaya</string>
   </property>
  </action>
  <action name="actionBorrame">
   <property name="icon">
    <iconset resource="../iconos.qrc">
     <normaloff>:/imagenes/iconos/botonera/segundo_plano.png</normaloff>:/imagenes/iconos/botonera/segundo_plano.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Segundo plano</string>
   </property>
  </action>
  <action name="actionMame">
   <property name="icon">
    <iconset resource="../iconos.qrc">
     <normaloff>:/imagenes/iconos/botonera/mame.png</normaloff>:/imagenes/iconos/botonera/mame.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Mame</string>
   </property>
  </action>
  <action name="actionactionVeleta">
   <property name="icon">
    <iconset resource="../iconos.qrc">
     <normaloff>:/imagenes/iconos/botonera/veleta.png</normaloff>:/imagenes/iconos/botonera/veleta.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Veleta</string>
   </property>
   <property name="toolTip">
    <string>Veleta</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="icon">
    <iconset resource="../iconos.qrc">
     <normaloff>:/imagenes/iconos/botonera/salida.png</normaloff>:/imagenes/iconos/botonera/salida.png</iconset>
   </property>
   <property name="text">
    <string>Sa&amp;lir</string>
   </property>
   <property name="toolTip">
    <string>Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F8</string>
   </property>
  </action>
  <action name="actionNuevo_Citrix">
   <property name="icon">
    <iconset resource="../iconos.qrc">
     <normaloff>:/imagenes/iconos/botonera/AD.png</normaloff>:/imagenes/iconos/botonera/AD.png</iconset>
   </property>
   <property name="text">
    <string>Nuevo Citrix</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Nuevo citrix 10.1.100.71&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionViejo_Citrix">
   <property name="icon">
    <iconset resource="../iconos.qrc">
     <normaloff>:/imagenes/iconos/botonera/AD.png</normaloff>:/imagenes/iconos/botonera/AD.png</iconset>
   </property>
   <property name="text">
    <string>Viejo_Citrix</string>
   </property>
   <property name="toolTip">
    <string>Viejo Citrix</string>
   </property>
  </action>
  <action name="actionLink">
   <property name="icon">
    <iconset resource="../iconos.qrc">
     <normaloff>:/imagenes/iconos/botonera/link.png</normaloff>:/imagenes/iconos/botonera/link.png</iconset>
   </property>
   <property name="text">
    <string>Abrir todos los Link </string>
   </property>
   <property name="toolTip">
    <string>Link</string>
   </property>
  </action>
  <action name="actionApp">
   <property name="icon">
    <iconset resource="../iconos.qrc">
     <normaloff>:/imagenes/iconos/botonera/app.png</normaloff>:/imagenes/iconos/botonera/app.png</iconset>
   </property>
   <property name="text">
    <string>Abrir todas las App</string>
   </property>
   <property name="toolTip">
    <string>App</string>
   </property>
  </action>
  <action name="actionVirtManager">
   <property name="icon">
    <iconset resource="../iconos.qrc">
     <normaloff>:/imagenes/iconos/botonera/virt-manager-icon.png</normaloff>:/imagenes/iconos/botonera/virt-manager-icon.png</iconset>
   </property>
   <property name="text">
    <string>VirtManager</string>
   </property>
   <property name="toolTip">
    <string>Virt-Manager</string>
   </property>
  </action>
  <action name="actionMailing">
   <property name="icon">
    <iconset resource="../iconos.qrc">
     <normaloff>:/imagenes/iconos/basedatos/mailing.png</normaloff>:/imagenes/iconos/basedatos/mailing.png</iconset>
   </property>
   <property name="text">
    <string>Mailing</string>
   </property>
   <property name="toolTip">
    <string>Mailing</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../iconos.qrc"/>
 </resources>
 <connections/>
</ui>
